<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>930</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stack">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="main_page">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="1" column="0">
           <layout class="QFormLayout" name="formLayout">
            <item row="1" column="1">
             <widget class="QPushButton" name="go_to_reg">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Регистрация заказчика</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QPushButton" name="make_order_button">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Оформить заказ</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="help_button">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Справка по странице</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QPushButton" name="oreder_view">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Посмотреть заказы</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>logo.png</pixmap>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="reg_page">
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="1">
           <widget class="QTableView" name="person_info"/>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="go_home_1">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>На главную</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="reg_button">
            <property name="font">
             <font>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Пройти регестрацию</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="order_page">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_3">
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <item row="0" column="0">
             <widget class="QComboBox" name="rest_choose">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>McDonald`s</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>KFC</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Burger King</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выбор ресторана</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QComboBox" name="category_choose">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <item>
               <property name="text">
                <string/>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Бургер</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Закуска</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Напиток</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLabel" name="label_4">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Выбор категории блюда</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QGridLayout" name="gridLayout_8">
              <item row="5" column="0">
               <widget class="QPushButton" name="position_5_pick">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QPushButton" name="position_2_pick">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QPushButton" name="position_3_pick">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QPushButton" name="position_1_pick">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QPushButton" name="position_4_pick">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="make_order">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Оформить заказ</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLabel" name="label_6">
              <property name="font">
               <font>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ваш заказ:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QTextBrowser" name="check">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QPushButton" name="go_home_2">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>На главную</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QProgressBar" name="cooking_proc">
              <property name="value">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="ready">
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Ваш заказ готов!</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="view_order_page">
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_12">
          <item row="0" column="0">
           <layout class="QFormLayout" name="formLayout_7">
            <property name="rowWrapPolicy">
             <enum>QFormLayout::DontWrapRows</enum>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLabel" name="label_15">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Все заказы:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QPushButton" name="go_home_3">
              <property name="font">
               <font>
                <pointsize>10</pointsize>
               </font>
              </property>
              <property name="text">
               <string>На главную</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QTableView" name="tableView"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
